tests/by-util/test_cp.rs

```diff
@@ -169,6 +168,32 @@ fn test_cp_multiple_files() {
     assert_eq!(at.read(TEST_HOW_ARE_YOU_DEST), "How are you?\n");
 }
 
+#[test]
+fn test_cp_multiple_files_with_nonexistent_file() {
+    let (at, mut ucmd) = at_and_ucmd!();
+    ucmd.arg(TEST_HELLO_WORLD_SOURCE)
+        .arg(TEST_NONEXISTENT_FILE)
+        .arg(TEST_HOW_ARE_YOU_SOURCE)
+        .arg(TEST_COPY_TO_FOLDER)
+        .fails();
```

Comment:
`.fails()` is a bit too general: What if we fail due to something unrelated?

Suggestion: Add something like `.stderr_contains(…);` in order to check for something that indicates that TEST_NONEXISTENT_FILE is indeed nonexistent.

Reply:
I will think about it, but error messages on Windows for non-existent files seem to have nothing in common with error messages on other platforms.

Reply:
I've added another commit which attempts to detect the specific error outputs.